app/models/display_ad.rb

```diff
@@ -36,6 +36,6 @@ def process_markdown
     #                                                         attributes: %w[href target src height width style]
     stripped_html = initial_html.html_safe # rubocop:disable Rails/OutputSafety
     html = stripped_html.delete("\n")
-    self.processed_html = MarkdownParser.new(html).prefix_all_images(html, 350)
+    self.processed_html = HtmlParser.new(html).prefix_all_images(350).html
```

Comment:
Nice, I like how this reads :-) In the interest of keeping our services folder clean, should it maybe be `Html::Parser`?

Reply:
+1 on this idea, so everything sits under `services/html`

Reply:
And change `MarkdownParser` to `Markdown::Parser` under `services/markdown`?

Reply:
Just kidding, that causes conflicts with `Redcarpet` 😅

Reply:
Updated!